### PR TITLE
Enabling CORS for HTTP API with Lambda proxy integration using SAM

### PR DESCRIPTION
**Pull request by Pallavi-D-Bhat**
*Description of changes:*
This pattern contains the source code and supporting files for a serverless application that you can deploy with the SAM CLI. Deploying the SAM file creates an HTTP API with Lambda proxy integration and CORS enabled.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jbesw** [comment]
The pattern metadata file does not validate - please fix and submit an updated PR.

**Comment by Pallavi-D-Bhat** [comment]
> The pattern metadata file does not validate - please fix and submit an updated PR.

I have updated the example-pattern.json file. Please verify

File: apigw-http-api-cors-sam/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+5. Note the outputs from the SAM deployment process. These contain the resource names and/or ARNs which are used for testing.
+
+## How it works
+
+An HTTP API is created that has ANY method with Lambda proxy integration. CORS headers  'Access-Control-Allow-Methods', 'Access-Control-Allow-Headers', 'Access-Control-Allow-Origin' are defined at the HTTP API level. If you configure CORS for an HTTP API, API Gateway automatically sends a response to preflight OPTIONS requests, even if there isn't an OPTIONS route configured for your API. For a CORS request, API Gateway adds the configured CORS headers to the response from an integration.
+
+## Testing
+
+To return CORS headers, your request must contain an 'origin' header. This will make the request to be from cross-origin.
```
Comment (mavi888) [inline apigw-http-api-cors-sam/README.md:45]: can you provide some examples of requests and responses that the reader might try and see.

File: apigw-http-api-cors-sam/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# HTTP API with CORS
+
+This pattern contains the source code and supporting files for a serverless application that you can deploy with the SAM CLI. Deploying the SAM file creates an HTTP API with Lambda proxy integration and CORS enabled.
```
Comment (mavi888) [inline apigw-http-api-cors-sam/README.md:3]: What are you achieving with this pattern? Can you elaborate more on why all the CORS conf is with allow all? What is the benefit of this configuration?

File: apigw-http-api-cors-sam/README.md (hunk 1 above)
Comment (mavi888) [reply apigw-http-api-cors-sam/README.md:45]: can you do this with curl? if so can you provide the CLI for this

**Comment by mavi888** [comment]
I will close this PR due to the lack of response from the author. 
Please reopen it when the comments are fixed

**Pull request closed without merging**